cms/djangoapps/contentstore/__init__.py

```diff
@@ -1,2 +0,0 @@
-""" module init will register signal handlers """
-import contentstore.signals
```

Comment:
@martynjames Can you put some light on why this is necessary here? With this line in `__init__.py`, I can't import the constant `SPLIT_TEST_COMPONENT_TYPE`.

This is just a temporary solution. I would like to get your opinion on the matter since I saw that you've added this as a part of #7313.